.github/workflows/rundoc.yml

```diff
@@ -0,0 +1,41 @@
+name: run-rundoc
+run-name: running rundoc on tutorials
+on:
+  schedule:
+    # This schedule triggers the workflow at 00:00 every Monday
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+jobs:
+  rundoc-ubuntu:
+    runs-on: pub-hk-ubuntu-24.04-ip
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3.2'
+          bundler-cache: true # do run 'bundle install' and cache
+          working-directory: .rundoc-workspace
+      - uses: buildpacks/github-actions/setup-pack@v5.7.1
+      # [CONFIG] here for the path of what we want to generate
+      - run: bundle exec rundoc ../docs/src/ruby/ruby_tutorial.md --on-success-dir ../docs/ruby --on-failure-dir ../docs/fail --force
+        working-directory: .rundoc-workspace
+        continue-on-error: true
+      - run: rm -rf ruby-getting-started
+        working-directory: docs/ruby
+        continue-on-error: true
+      # [CONFIG] then here move and rename README.md to the appropriate place
+      - run: echo "NOW=$(date +'rundoc-linux/%s')" >> $GITHUB_ENV
+      - run: echo "HUMAN_DATE=$(date +'update tutorials - %Y %b %d')" >> $GITHUB_ENV
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          branch: ${{ env.NOW }}
+          base: main
+          commit-message: run rundoc
+          title: ${{ env.HUMAN_DATE }}
+          body: "generated by the [run-rundoc linux github action](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
```

Comment:
Could you add an "Enable PR auto-merge" step below this one, like:
https://github.com/heroku/cnb-builder-images/blob/ca589d528e7510bbad18244e201437a59ba64b6f/.github/workflows/update-lifecycle.yml#L74-L78

(The automated PRs will still need review, but it saves a click after the review)